common/chat/contacts.ts

```diff
@@ -102,6 +104,7 @@ const getInstructorsForPupilSubcourses = async (pupil: User): Promise<SubcourseC
                         allowChatContactParticipants: true,
                         subcourse_participants_pupil: { some: { pupilId: pupil.pupilId } },
                         cancelled: false,
+                        lecture: { every: { start: { gte: today } } },
```

Comment:
The inverse of every(start <= today) is some(start >= today) ;) I think it should be some instead of every